components/audit_table/index.js

```diff
@@ -15,6 +16,7 @@ function mapStateToProps(state) {
         currentUser: getCurrentUser(state),
         getUser: (userId) => getUser(state, userId),
         getByName: (channelName) => getChannelByName(state, channelName),
+        getDirectTeammate: (channelId) => getDirectTeammate(channelId, state),
```

Comment:
This isn't greatsince `mapStateToProps` is called whenever anything in the redux state changes, and it will pass a new function in each time that happens which will cause the component to re-render far too often, but that's an existing issue because of the other props, so I don't think there's anything we can change here.

Reply:
sorry for adding to the problem but at least we made steps in the utils 🙈